listenbrainz/mbid_mapping/mapping/typesense_index.py

```diff
@@ -80,11 +117,15 @@ def build_index():
                 documents.append(document)
 
                 if len(documents) == BATCH_SIZE:
-                    client.collections[COLLECTION_NAME].documents.import_(documents)
+                    client.collections[collection_name].documents.import_(documents)
                     documents = []
 
-                if i and i % 100000 == 0:
-                    print(i)
+                if i and i % 1000000 == 0:
+                    log("typesense index: Indexed %d rows" % i)
 
             if documents:
-                client.collections[COLLECTION_NAME].documents.import_(documents)
+                client.collections[collection_name].documents.import_(documents)
+
+    log("typesense index: indexing complete. waiting for background tasks to finish.")
```

Comment:
just out of interest, what background tasks are happening at this point?

Reply:
Dunno, its not specified in the docs. But if I terminate right away, I get nasty sounding error messages.